### PR TITLE
Bulk JSONs

### PR DESCRIPTION
**Pull request by PrivacySecured**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [X] The file is in the `domains` folder and is in the JSON format.
- [X] The file's name is all lowercased and alphanumeric. 
- [X] You have completed your website. 
- [X] The website is reachable.  
- [X] The CNAME record doesn't contain `https://` or `/`.  
- [X] There is sufficient information at the `owner` field.  

## Website Link/Preview

https://privacysecured.github.io/home - I am currently finishing up the page as I make this PR :D

### CODE REVIEW

File: domains/l.ps.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "owner": {
+    "username": "PrivacySecured",
+	"email": "https://email.ps.is-a.dev/", 
+    "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.VoJDQtbSUW3kYsQ0IuIE4CTIomV-e8GVUO2wS0fZp_-DkO6QltgcnKL_mAqMH5Salut7IGMZy48HExbambDrl11jQWwYHBAqqyAuAlxsqnUh83KGY3UKYfQzA8eRkdCPTth1pxvyXfUFnAnjEtmJLMNTSCds-J8fmcIkNEu-xT2UMEjxPQkoN8sJ7EA8YTHn4t3078D8tMr3AV5DJ1rg0QnuFwjra5FbjuJGHeYenEzS-Tdha7LTUyTEbKrdqdxYNpN4pv45sNrVpsDZPt3sR1CgOnXNSsfVwjUm-5DRweNquJHUxSpNZblJnbWBErWyg9_NZvISrClpJZ4We48oDg.YdsO9C1ulWhtaJYei-z-Zg.8p4HE12styyP4JSTFrlLWMmfIUAOw1kDQWrAcqSOlqbbtujr4Ww-QyBuUBIQojGvuFKs3jflF5tuvIjOgU8hTOWxB2uS4KItHwpIsFeQo9i1OHSTzj0glP2ly-XMY1Wi.99kVL134lF7knduFKnqfGQ",
+    "notes": "I rarely go on my emails. If I do its because I needed a verification code."
+  },
+  "record": {
+    "CNAME": "cname.short.io"
```
Comment (Stef-00012) [inline domains/l.ps.json:9]: Can you show a preview of your site?
I can not reach it because it has no SSL and has HSTS enabled

**Review by Stef-DP — DISMISSED** [review]

**Comment by Stef-00012** [comment]
bbtw if you don't want to show your email, you can just set the field as an empty string (`""`), doesn't rlly matter since the mail is also encrypted inside the OWL

File: domains/l.ps.json (hunk 1 above)
Comment (Newfies) [reply domains/l.ps.json:9]: https://privacysecured.github.io/home - gonna add instructional stuff to it later but it is complete

Comment (PrivacySecured) [reply domains/l.ps.json:9]: https://privacysecured.github.io/home/

**Comment by PrivacySecured** [comment]
@Stef-00012 - Updated everything, please accept this request

**Pull request merged**